src/main/java/amazin/controller/BookController.java

```diff
@@ -0,0 +1,82 @@
+package amazin.controller;
+
+import java.util.Optional;
+
+import javax.validation.Valid;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Controller;
+import org.springframework.ui.Model;
+import org.springframework.validation.BindingResult;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.ModelAttribute;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.SessionAttributes;
+
+import amazin.model.Book;
+import amazin.service.BookService;
+
+@Controller
+@SessionAttributes("book")
+public class BookController {
+    public static final String VIEW_CREATE_BOOK = "create-book";
+    public static final String VIEW_UPDATE_BOOK = "update-book";
+    public static final Object MODEL_ATTRIBUTE_BOOk = "book";
```

Comment:
BOOk? BOOK? book? boOK! 🙉

Reply:
-> capitalization mixing needed to be fixed here for variable MODEL_ATTRIBUTE_BOOk

Reply:
Fixed. Good catch.